### PR TITLE
Rename kogitobot to kiebot

### PR DESCRIPTION
**Pull request by radtriste**

### CODE REVIEW

**Comment by radtriste** [comment]
@bot close

**Comment by radtriste** [comment]
@bot open

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# kogibot
+# kiebot
 
-![Kogibot - Java 11](https://github.com/spolti/kogibot/actions/workflows/main-merge-java-11.yml/badge.svg) 
-![Kogibot - Java 16](https://github.com/spolti/kogibot/actions/workflows/main-merge-java-16.yml/badge.svg)
-![Kogibot - Java 17](https://github.com/spolti/kogibot/actions/workflows/main-merge-java-17.yml/badge.svg)
+![Kiebot - Java 11](https://github.com/spolti/kiebot/actions/workflows/main-merge-java-11.yml/badge.svg)
```
Comment (spolti) [inline README.md]: for the badges we can use the url from the current job, pull_request I guess

Comment (radtriste) [reply README.md]: yep

**Comment by radtriste** [comment]
@bot retest

**Comment by spolti** [comment]
@bot help

File: README.md (hunk 1 above)
Comment (radtriste) [reply README.md]: updated

**Review by spolti — APPROVED** [review]

**Comment by radtriste** [comment]
not implemented yet :)

**Comment by radtriste** [comment]
*activated. It needs testing first :D

**Pull request merged**